torchtune/utils/_checkpointing/_checkpointer.py

```diff
@@ -381,12 +382,15 @@ def load_checkpoint(self) -> Dict[str, Any]:
             del state_dict
             gc.collect()
 
-        converted_state_dict[utils.MODEL_KEY] = convert_weights.hf_to_tune(
-            merged_state_dict,
-            num_heads=self._config["num_attention_heads"],
-            num_kv_heads=self._config["num_key_value_heads"],
-            dim=self._config["hidden_size"],
-        )
+        if self._model_type == ModelType.PHI3_MINI:
```

Comment:
Does capitalization change this? B/c in the config it's `PHI_MINI` and in the enum it's `phi_mini`

Reply:
Yup, it matters!